### ISSUE TITLE
[Feature Request] Simplify No. to Read

### ISSUE THREAD

**Issue by saltsoftdrink** [Pool-Of-Tears/GreenStash#86]
### Checklist

- [X] This issue contains only one feature request. I will open one issue for every feature I want to request.
- [X] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo.
- [ ] This feature is merely an UI/UX update.
- [X] This feature is not going to conflict with many of the existing features/options.

### Is your feature request related to a problem? Please describe.

It's so confusing that how much zeros are added in amount section

### Describe the solution you'd like

If you'd try to add automatically commas between the numbers it will become much more easier to understand the number

### How will you/everyone benefit from this feature?

It will be easier to read number instead of counting how much numbers are there

**Comment by starry-shivam** [comment]
Good idea. I remember I once tried to implement this, but for some reason, I dropped the idea. I'll try it again in upcoming releases. Thanks
